pages/common/miniserve.md

```diff
@@ -0,0 +1,17 @@
+# miniserve
+
+> Simple HTTP file server CLI.
+> For when you really just want to serve some files over HTTP right now!.
+> More information: <https://github.com/svenstaro/miniserve>.
+
+- Serve a directory:
+
+`miniserve {{path/to/directory}}`
+
+- Serve a single file:
+
+`miniserve {{path/to/source_file}}`
```

Comment:
```suggestion
`miniserve {{path/to/file}}`
```

Reply:
Thank you for good suggestion.
It's looks so good.